canonical-websites/www.ubuntu.com
# Update esm 12 04

## Done

- Updated end of ESM for 12.04 to 2019 on /about/release-cycle

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/about/release-cycle](http://0.0.0.0:8001/about/release-cycle)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- In the charts, see that 12.04 esm support ends in april 2019 as on [data doc](https://docs.google.com/spreadsheets/d/1KIuOAa620yseE0uhIl67u2SlS0bWlPI_d98bvYM2Yl8/edit#gid=1712489059)

Fixes #4549
